src/main/java/com/ldtteam/blockui/BOGuiGraphics.java

```diff
@@ -184,4 +185,9 @@ public void popMvPose()
         RenderSystem.getModelViewStack().popMatrix();
         RenderSystem.applyModelViewMatrix();
     }
+
+    public static double getAltSpeedFactor()
```

Comment:
What is this?

Reply:
Temporary thing to allow fast zoom/drag/scroll when holding alt, cuz i have apparently high dpi monitor now and it was super slow

Reply:
xD